cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/TestCommand.java

```diff
@@ -326,6 +336,11 @@ public void execute() {
                     "flag is not set");
         }
 
+        if (!project.buildOptions().cloud().isEmpty() && project.buildOptions().codeCoverage()) {
+            this.outStream.println("WARNING: Code coverage generation is not supported with Ballerina cloud test");
```

Comment:
Let's test how `--cloud=k8s` is handled too. 
We have to issue a warning saying tests are only supported with `--cloud=docker`

Reply:
Will work on that. After issuing the warning, should we abort the execution?

Reply:
Either we need abort or skip the cloud flag and run normal tests IMO